app/templates/components/forms/orders/order-form.hbs

```diff
@@ -136,6 +136,11 @@
                 <label for="payment_mode"><img src="/images/payment-logos/paypal.png" alt="paypal"></label>
               </div>
             {{/if}}
+            {{#if data.event.canPayByOmise}}
```

Comment:
If all other radio buttons are inside a `field` div, this one should be inside it too.